ara/exec/exec_error_domain.h

```diff
@@ -1,4 +1,40 @@
 #ifndef ARA_EXEC_ERR_DOMAIN_H_
 #define ARA_EXEC_ERR_DOMAIN_H_
 
+#include "error_code.h"
+
+namespace ara {
+namespace exec {
+
+enum class ExecErrc : ara::core::ErrorDomain::CodeType {
+    kGeneralError = 1,
+    kInvalidArguments,
+    kCommunicationError,
+    kMetaModelError,
+    kCancelled,
+    kFailed
+};
+
+class Exception {};
```

Comment:
Rozumiem ze w dokumentacji ExecException dziedziczy po ara::core::Exception ale ten nie jest nigdzie zdefiniowany. Po samym wystapieniu tego wyjatku domyslam sie ta klasa i tak jest tozsama ze zwyklym std::exception i to po nim bym dziedziczył.

Reply:
ta klasa jest zdefiniowana w drugim PR- stąd mozna ją usunac :)